test/karma.conf.js

```diff
@@ -86,7 +86,7 @@ module.exports = (config) => {
     browserDisconnectTimeout: 20000,
     browserNoActivityTimeout: 240000,
     captureTimeout: 120000,
-    browsers: ['Chrome_1024x768'],
+    browsers: process.env.TRAVIS ? ['ChromeHeadlessCI'] : ['ChromeHeadlessCISandbox'],
```

Comment:
This shouldn't be necessary. This never runs in Travis

Reply:
Fixed